### DISCUSSION TITLE
Need customized column width for Expansion column

### DISCUSSION THREAD

**Discussion by manlasgit** [KevinVandy/material-react-table#991] [Ideas]
The width for expansion column looks large, Please provide customized width
![mrt-git](https://github.com/KevinVandy/material-react-table/assets/95064789/e88c632e-26aa-4cf2-bcd9-fd70e8d38d13)

**Comment by KevinVandy** [comment]
The docs show you how to do this in multiple places https://www.material-react-table.com/docs/guides/expanding-sub-rows#customize-expand-column
